falcon/util/sync.py

```diff
@@ -212,7 +228,7 @@ def async_to_sync(coroutine, *args, **kwargs):
     return loop.run_until_complete(coroutine(*args, **kwargs))
 
 
-def runs_sync(coroutine):
+def runs_sync(coroutine: Callable[..., Awaitable[Any]]) -> Callable[..., Any]:
```

Comment:
isn't this awaitable[Result] and Callable[..., Result] ?

Reply:
Changed